src/chapters.py

```diff
@@ -0,0 +1,20 @@
+import mistune
+
+renderer = mistune.Renderer()
+markdown = mistune.Markdown(renderer=renderer)
+
+
+def validate_chapter(lang, year, chapter):
+    # TODO
+    return True
+
+
+# TODO: Should this be cached in memory to prevent having to repeatedly
+# open and read the file? It shouldn't lock, I'm just not sure how well
+# this will perform in python.
```

Comment:
As mentioned in the comment, I am nervous of the performance of this. It may be nothing, I'm just not very experienced with python in production. This shouldn't lock if concurrent threads are reading files, and I suppose it's a tradeoff between CPU (opening and reading) versus Memory (keeping the entire contents of the file in memory) when it's all running on app engine.